diff --git a/node_modules/react-devtools-core/.bun-tag-d7dca9fadb831554 b/.bun-tag-d7dca9fadb831554
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/dist/backend.js b/dist/backend.js
index 479bdbf10077a6e44e95aefe2016af8ce91ced74..c21e94374278694939c6a1fa056cf1d8305e857b 100644
--- a/dist/backend.js
+++ b/dist/backend.js
@@ -14952,6 +14952,10 @@ function installHook(target, maybeSettingsOrSettingsPromise) {
       patchConsoleForErrorsAndWarnings();
     }).catch(function () {
       targetConsole.error("React DevTools failed to get Console Patching settings. Console won't be patched and some console features will not work.");
+      var settings = maybeSettingsOrSettingsPromise;
+      hook.settings = settings;
+      hook.emit('settingsInitialized', settings);
+      patchConsoleForErrorsAndWarnings();
     });
   }
   Object.defineProperty(target, '__REACT_DEVTOOLS_GLOBAL_HOOK__', {
diff --git a/package.json b/package.json
index c8facd1a24d4636f778c7081ae8408a0398bceb2..136b741ed4c80ca83fa0ab159f811188550e4ed6 100644
--- a/package.json
+++ b/package.json
@@ -4,6 +4,9 @@
   "description": "Use react-devtools outside of the browser",
   "license": "MIT",
   "main": "./dist/backend.js",
+  "exports": {
+    ".": "./dist/backend.js"
+  },
   "repository": {
     "type": "git",
     "url": "https://github.com/facebook/react.git",
